app/forms/email_form.py

```diff
@@ -4,13 +4,15 @@
 
 from app.forms import error_messages
 from app.forms.questionnaire_form import map_subfield_errors
+from app.forms.validators import EmailTldCheck
 
 
 class EmailForm(FlaskForm):
     email = StringField(
         validators=[
             InputRequired(error_messages["MANDATORY_EMAIL"]),
             Email(error_messages["INVALID_EMAIL_FORMAT"]),
+            EmailTldCheck(error_messages["INVALID_EMAIL_FORMAT"]),
```

Comment:
A minor issue, since we have 2 validators with the same error message, when an email fails both validators then 2 error messages are shown but because they are the same, it looks odd. Ideally, it would display 1 or 2 messages there are unique to the error. Need confirmation about expected behaviour.

To recreate, use an email that fails WTForms and our new check like `m@@c.o` which fails because of the double @ sign and because of the tld.

![image](https://user-images.githubusercontent.com/35296336/109677175-51deaa00-7b71-11eb-83b4-6c542d934460.png)

Reply:
Good spot. I have moved the TLD validation to before the WTForms validation, and validation chain is now stopped if it fails TLD validation.